lib/assessment/assess.py

```diff
@@ -34,6 +34,14 @@ def grade(code, prompt, rubric, examples=[], api_key='', llm_model='gpt-4', num_
   else:
     logging.info("Using set OPENAI_API_KEY")
 
+  # Validate example key concepts against rubric.
+    for ex in examples:
+      rubric_key_concepts = list(set(row['Key Concept'] for row in csv.DictReader(rubric.splitlines())))
+      example_key_concepts = list(set(row['Key Concept'] for row in csv.DictReader(ex[1].splitlines(), delimiter="\t")))
+      if rubric_key_concepts != example_key_concepts:
+        logging.error("Rubric and example key concepts do not match.")
```

Comment:
my experience so far has been that when I hit these kinds of errors, my next step is to go in and change the error message to print out more details about what the mismatch was. would you be up for adding those details now so we can skip that step? for example: https://github.com/code-dot-org/aiproxy/blob/ba49fbf04175b0a9457aa8427dc3f9eaa08115a7/lib/assessment/grade.py#L264

Reply:
Done!